app/assets/stylesheets/procedure_champs_editor.scss

```
@@ -104,7 +102,7 @@
         border-bottom: 1px solid $border-grey;
 
         &.head {
-          border-bottom: 1px solid #D4E5F5;
+          border-bottom: 1px solid #CCCCCC;
```

si on veut vraiment chipoter, ça pourrait aussi être 

```suggestion
          border-bottom: 1px solid var(--border-default-grey)`
```